dataverse-webapp/src/main/webapp/dataset-versions.xhtml

```diff
@@ -415,8 +415,73 @@
                     </p:column>
                 </p:dataTable>
             </div>
+        </ui:fragment>
+
+        <ui:fragment rendered="#{datasetVersionsTab.versionsDifferenceForDialog.isNotEmpty() and datasetVersionsTab.isUserUnderEmbargo()}">
```

Comment:
IMO instead of duplicating the whole thing - we could make calculation of differences dependent on extra parameter (adding extra constructor in `DatasetVersionDifference`. 
```
public DatasetVersionDifference(DatasetVersion newVersion, DatasetVersion originalVersion, boolean includeFileDifferences) {
```
That way we could manipulate whether we want to display differences in files or not in java code:
```
    public void updateVersionDiffForModalFromSelected() {
...
        updateVersionDiffForDialog(new DatasetVersionDifference(newVersion, oldVersion, doWeWantToIncludeDifferencesInFiles));
...
If we would move the code to java it would be easier to write some tests or make summary column in versions listing consistent with dialog (not the scope of current task).

We are also exposing indirectly whether there are some changes in files. If text in lines below will not be displayed and there will be no changes in metadata then that means that there are some changes in files
```
        <ui:fragment rendered="#{datasetVersionsTab.versionsDifferenceForDialog.isEmpty()}">
            <p class="help-block">
                <h:outputFormat value="#{bundle['file.viewDiffDialog.dialog.noDiff']}">
                    <f:param value="#{datasetVersionsTab.versionsDifferenceForDialog.originalVersion.semanticVersion}"/>
                    <f:param value="#{datasetVersionsTab.versionsDifferenceForDialog.newVersion.semanticVersion}"/>
                </h:outputFormat>
            </p>
        </ui:fragment>
```